### PR TITLE
Emoji html fix

### PR DESCRIPTION
**Pull request by dougfelt**
Bunch of emoji changes.

- Update IC images from behdad's region_flags repo.
- Support a few more flags by default
- Alias a few territories to use larger country flags
- Use 'unknown flag' for regional indicator pairs we don't support
- Add tool to generate html comparison file of emoji-named images.

### CODE REVIEW

File: generate_emoji_html.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+def _get_name(key_tuple):
+  CELL_PREFIX = '<td class="name">'
+  if len(key_tuple) != 1:
+    name = ''
+  else:
+    cp = key_tuple[0]
+    if cp in unicode_data.proposed_emoji_cps():
+      name = '(proposed) ' + unicode_data.proposed_emoji_name(cp)
+    else:
+      name =unicode_data.name(cp, '(error)')
```
Comment (jamesgk) [inline generate_emoji_html.py]: whitespace

File: generate_emoji_html.py [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+          '<img src="%s">' % key_to_filename[tuple([key])]
+          for key in key_tuple
+          if tuple([key]) in key_to_filename)
+    name = _get_name(key_tuple)
+    lines.append('<tr><td><img src="%s"><td class="desc">'
+                 '%s<td class="name">'
+                 '%s' % (
+        key_to_filename[key_tuple], key_string, name))
+  return '\n  '.join(lines) + '\n<table>'
+"""
```
Comment (jamesgk) [inline generate_emoji_html.py]: Should this be left in?

File: generate_emoji_html.py [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  fails = []
+  result = {}
+  expect_re = re.compile(r'%s([0-9A-Fa-f_]+).%s' % (prefix, ext))
+  for f in sorted(glob.glob(path.join(image_dir, '*.%s' % ext))):
+    filename = path.basename(f)
+    m = expect_re.match(filename)
+    if not m:
+      if filename.startswith('unknown_flag.'):
+        continue
+      fails.append('"%s" did not match: "%s"' % (expect_re, filename))
```
Comment (jamesgk) [inline generate_emoji_html.py]: I think what you want here is `expect_re.pattern`

**Comment by jamesgk** [comment]
LGTM

File: generate_emoji_html.py (hunk 2 above)
Comment (dougfelt) [inline generate_emoji_html.py]: Urm... no.

File: generate_emoji_html.py (hunk 3 above)
Comment (dougfelt) [inline generate_emoji_html.py]: Yes, thanks.

**Pull request merged**